tests/functional/amo_sign_up.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'require',
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/fx-desktop'
+], function (intern, registerSuite, require, FunctionalHelpers, FxDesktopHelpers) {
+  var config = intern.config;
+  var PAGE_URL = config.fxaContentRoot + 'signin?context=fx_desktop_v1';
```

Comment:
@vbudhram - I was thinking about how to properly test this. I think I'd start with these tests in [oauth_sign_up.js](https://github.com/mozilla/fxa-content-server/blob/0c84ed2695abf0411c875aa499f61be38e191464/tests/functional/oauth_sign_up.js#L112), using `openFxaFromRp`. `openFxaFromRp` accepts a `urlSuffix`, which I believe was a bad but workable idea on my part to not modify too many tests too quickly. If you call `openFxaFromRp` with a `urlSuffix` of `migration=amo`, I think that should open FxA in the proper state.